### PR TITLE
feat: Send telemetry about failed COPY FROMs

### PR DESCRIPTION
**Pull request by irmatov**
Closes cloudquery/cloudquery-issues#374

### CODE REVIEW

**Comment by irmatov** [comment]
https://github.com/cloudquery/cloudquery/pull/1050/ is a continuation on the core side.

File: provider/execution/execution.go [hunk 1]
```diff
@@ -301,34 +301,42 @@ func (e TableExecutor) resolveResources(ctx context.Context, meta schema.ClientM
...
+	diags = diags.Add(diag.TelemetryFromError(err, diag.BulkInsertFailed))
 	// Setup diags, adding first diagnostic that bulk insert failed
-	diags := diag.Diagnostics{}.Add(ClassifyError(err, diag.WithType(diag.DATABASE), diag.WithSummary("failed bulk insert on table %q", e.Table.Name)))
+	diags = diags.Add(ClassifyError(err, diag.WithType(diag.DATABASE), diag.WithSummary("failed bulk insert on table %q", e.Table.Name)))
 	// Try to insert resource by resource if partial fetch is enabled and an error occurred
 	partialFetchResources := make(schema.Resources, 0)
+	var failed error
 	for id := range resources {
 		if err := e.Db.Insert(ctx, e.Table, schema.Resources{resources[id]}, shouldCascade); err != nil {
+			failed = err
```
Comment (disq) [inline provider/execution/execution.go:329]: would it make sense to also capture the percentage of resources failed? or at least a bool to tell us if it failed for "all resources" or just a few.

Comment (irmatov) [reply provider/execution/execution.go:330]: Added appropriate information, take a look, please.

Comment (bbernays) [reply provider/execution/execution.go:330]: Isn't that information able to be calculated based on the final message?

Comment (irmatov) [reply provider/execution/execution.go:330]: I just added it based on Kemal's comment. :)

**Review by disq — APPROVED** [review]
LGTM

**Pull request merged**